src/QsCompiler/TextProcessor/QsExpressionParsing.fs

```diff
@@ -695,29 +617,27 @@ let private lambda =
 
 // processing terms of operator precedence parsers
 
-type private MissingMode =
-    | NoMissing
-    | AllowMissing
-
-let private termParser missingMode tupleExpr =
-    // IMPORTANT: any parser here needs to be wrapped in a term parser, such that whitespace is processed properly.
-    [
-        attempt newArray
-        attempt lambda // needs to be before unitValue
-        if missingMode = AllowMissing then missingExpr // needs to be after lambda but before identifier
-        attempt unitValue
-        attempt callLikeExpr // needs to be after unitValue
-        attempt itemAccessExpr // needs to be after callLikeExpr
-        attempt valueArray // needs to be after arryItemExpr
-        attempt tupleExpr // needs to be after unitValue, arrayItemExpr, and callLikeExpr
-        attempt pauliLiteral
-        attempt resultLiteral
-        attempt numericLiteral
-        attempt boolLiteral
-        attempt stringLiteral
-        attempt identifier // needs to be at the very end
-    ]
-    |> choice
-
-qsExpression.TermParser <- valueTuple expr |> termParser NoMissing
-qsArgument.TermParser <- valueTuple argument |> termParser AllowMissing
+let private chainPostfix p op =
```

Comment:
Consider adding a comment to this function :)

Reply:
Added.

Reply:
Thanks @samarsha!